BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -12,23 +12,49 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
         {
             if (isDirectory)
             {
-                if (files.Any(f => f.Contains(Path.Combine("aacs", "VTKF000.AACS")))
-                    && files.Any(f => f.Contains(Path.Combine("AACS", "CPSUnit00001.cci"))))
+                if (files.Any(f => f.Contains(Path.Combine("AACS", "MKBROM.AACS"))))
```

Comment:
If the statements are the same, can you make this a single `if` statement with an or instead?

Reply:
You can leave a comment next to each part of the statement if you want to mark which is BD and which is HD-DVD